### DISCUSSION TITLE
Open popover programatically

### DISCUSSION THREAD

**Discussion by cubecleveland** [ariakit/ariakit#968] [Questions] [answered]
Hello, sorry for the silliy question but i couldnt fins it in the docs.
Is there anyway to leave open and close the popover programatically ?

**Comment by diegohaz** [comment]
Yes! You can call `popover.show()`, where `popover` is the state object returned by `usePopoverState()`.

**Comment by cubecleveland** [comment, marked as answer]
Thanks so much!

To follow up can i initiate a dialog open state right of the bet ? so the modal will be opened on page load ?

**Reply by diegohaz** [reply]
Yeah! You can do `useDialogState({ visible: true })`. The same works for popover, menu etc.
